packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlock.stories.tsx

```diff
@@ -1,19 +1,20 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react/'
-import CustomHTMLBlock from './CustomHTMLBlockStyle'
+import CustomHTMLBlockStyle from './CustomHTMLBlockStyle'
+import CustomHTMLBlock from './CustomHTMLBlock'
 
 storiesOf('Atoms/CustomHTMLBlock', module)
   .addDecorator(storyFn => (
     <div style={{ padding: '40px 10px' }}>{storyFn()}</div>
   ))
   .add('default', () => (
-    <CustomHTMLBlock>
+    <CustomHTMLBlockStyle>
```

Comment:
why not just use CustomHTMLBlock here?

Reply:
this construction with children is not supported by the `CustomHTMLBlock`, just by `CustomHTMLBlockStyle`